spec/facebook/messenger/subscriptions_spec.rb

```diff
@@ -16,13 +16,14 @@
     context 'with a successful response' do
       before do
         stub_request(:post, subscribed_apps_url)
-          .with(query: {
+          .with(headers: { 'Content-Type' => 'application/json' },
+                body: {
                   access_token: access_token,
                   subscribed_fields: subscribed_fields
-                })
+                }.to_json)
           .to_return(
             body: JSON.dump('success' => true),
-            status: 200,
+            status: 400,
```

Comment:
Being a successful response, shouldn't this status code remain `HTTP 200 OK`? Does Facebook respond with  `HTTP 400 Bad Request` for this?